service/update.go

```diff
@@ -56,10 +56,31 @@ func (service *UpdateServiceImp) StoreConceptUpdate(ctx context.Context, bytes [
 	return &updateDbo.ID, nil
 }
 
-func (service *UpdateServiceImp) GetConceptUpdates(ctx context.Context, conceptId string, page int, size int) (*model.Updates, int) {
+func (service *UpdateServiceImp) GetConceptUpdates(ctx context.Context, conceptId string, page int, size int, sortBy string, sortOrder string) (*model.Updates, int) {
 	query := bson.D{}
 	query = append(query, bson.E{Key: "resourceId", Value: conceptId})
-	databaseUpdates, err := service.ConceptsRepository.GetConceptUpdates(ctx, query, page, size)
+
+	// Set default sort by column to "datetime"
+	sortByCol := "datetime"
+	switch sortBy {
+	case "name":
+		sortByCol = "person.name"
+	case "email":
+		sortByCol = "person.email"
+	}
+
+	// Map sortOrder string to integer value
+	var sortOrderInt int
+	switch sortOrder {
+	case "asc":
+		sortOrderInt = 1
+	case "desc":
+		sortOrderInt = -1
+	default:
+		sortOrderInt = -1 // default to descending order
+	}
```

Comment:
Kanskje bare korte denne ned på samme måte som den foran?
```suggestion
	sortOrderInt := -1
	if sortOrder == "asc" {
		sortOrderInt = 1
	}
```